trino/sqlalchemy/dialect.py

```diff
@@ -237,6 +237,7 @@ def get_table_comment(self, connection: Connection, table_name: str, schema: str
         except error.TrinoQueryError as e:
             if e.error_name in (
                 error.NOT_FOUND,
+                error.NOT_SUPPORTED,
```

Comment:
I think the right solution is fixing query at `SELECT "comment" FROM {properties_table}`. 
It should be `SELECT comment FROM system.metadata.table_comments` in my opinion.

Reply:
@ebyhr that would require the user to also have access to the `system` catalog, so it might not be the best idea from security/permission management point of view

Reply:
@ms32035 First of all, `$properties` table is specific to Hive and Iceberg connector. We shouldn't use such tables.

Reply:
So what is the suggested approach?

Reply:
We also have `information_schema.tables.table_comment` I believe?

Reply:
Yes, but it's a hidden column for MySQL compatibility. Since JDBC driver uses `system.jdbc.tables` to get table metadata, I would recommend using `system.metadata.table_comments`.